pyproject.toml

```diff
@@ -57,6 +57,7 @@ pgvector = {version = "^0.1.6", optional = true}
 psycopg2-binary = {version = "^2.9.5", optional = true}
 boto3 = {version = "^1.26.96", optional = true}
 pyowm = {version = "^3.3.0", optional = true}
+wrapt = "^1.15.0"
```

Comment:
what is this used for? does this really need to be a requirement for the package or is more dev dependencies?